openassessment/xblock/openassessmentblock.py

```diff
@@ -611,7 +611,8 @@ def student_view(self, context=None):  # pylint: disable=unused-argument
             "title": self.title,
             "xblock_id": self.get_xblock_id(),
         }
-        template = get_template("base.html")
+        # template = get_template("legacy/oa_base.html")
```

Comment:
Is there a reason to leave in the commented out code?

Reply:
Nope, it was a mistake.